src/main/java/ProjectIndexer.java

```diff
@@ -27,10 +27,17 @@ public ProjectIndexer(Arguments arguments, Emitter emitter) {
     }
 
     public void index() {
+		// Because Paths.get(...).toAbsolutePath() keeps the '.' when '-projectRoot .'
+		Path projectRootPath = Paths.get(arguments.projectRoot).toAbsolutePath();
+		String projectRoot = projectRootPath.toString();
+		if (projectRootPath.getFileName().toString().equals(".")) {
+			projectRoot = projectRoot.substring(0, projectRoot.lastIndexOf('.'));
+		}
+
         emitter.emitVertex("metaData", Util.mapOf(
                 "version", "0.4.0",
                 "positionEncoding", "utf-16",
-                "projectRoot", String.format("file://%s", Paths.get(arguments.projectRoot).toAbsolutePath().toString()),
```

Comment:
If you just use `getCanonicalPath` instead does it still have the same issue?

Reply:
Good catch, that works well :+1: